lib/fog/aliyun/models/compute/security_group.rb

```diff
@@ -4,82 +4,83 @@ module Fog
   module Compute
     class Aliyun
       class SecurityGroup < Fog::Model
-        # {"SecurityGroup"=>[{"CreationTime"=>"2015-08-10T03:21:02Z", 
-        #   "SecurityGroupId"=>"sg-25mexinlu", "Description"=>"", "SecurityGroupName"=>"bosh", 
-        #   "VpcId"=>"vpc-25mj6mguq"}}
-
-        identity  :id,                    :aliases =>"SecurityGroupId"
-
-        attribute :name,                  :aliases => "SecurityGroupName"
-        attribute :description,           :aliases => "Description"
-        attribute :created_at,            :aliases => "CreationTime"
-        attribute :vpc_id,                :aliases => "VpcId"
+        identity  :id,           :aliases =>"SecurityGroupId"
+        attribute :name,         :aliases => "SecurityGroupName"
+        attribute :description,  :aliases => "Description"
+        attribute :created_at,   :aliases => "CreationTime"
+        attribute :vpc_id,       :aliases => "VpcId"
 
         def vpc
-            requires :vpc_id
-            $vpc=Fog::Compute::Aliyun::Vpcs.new(:service=>service).all('vpcId'=>vpc_id)[0]
+          requires :vpc_id
+          $vpc=Fog::Compute::Aliyun::Vpcs.new(:service=>service).all('vpcId'=>vpc_id)[0]
```

Comment:
It seems like I forgot to clickc comment button yesterday. Define a global variable just because when save vswitch, it need the vpc. And should I change  vpc methods in other models? Should I fix the $vpc in vpcs collection?